dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops

```diff
@@ -0,0 +1,772 @@
+
+ERROR_ASSET_INVALID_PRODUCT_EDITION=CWWKF1256E: The asset {0} cannot be installed to {1} {2} edition because it only applies to the editions {3}. Use the "find" action of the installUtility command to retrieve a list of assets that are applicable to {4} {5} edition. 
+ERROR_ASSET_INVALID_PRODUCT_EDITION.explanation=The asset is not applicable to the edition of the installed product.
+ERROR_ASSET_INVALID_PRODUCT_EDITION.useraction=Install assets that are supported by the product edition.
+
+ERROR_ASSET_INVALID_PRODUCT_INSTALLTYPE=CWWKF1257E: The {0} asset cannot be downloaded or installed with the installUtility command because the product was installed by {1}, and the asset applies only to {2} installations. Use the installUtility find action to retrieve a list of assets that you can download or install with the command.
+ERROR_ASSET_INVALID_PRODUCT_INSTALLTYPE.explanation=The installUtility command cannot download or install the asset to your Liberty installation because the asset applies only to a different installation type.
+ERROR_ASSET_INVALID_PRODUCT_INSTALLTYPE.useraction=Use the installUtility find action to retrieve a list of assets that you can download or install with the command. For more information, see the product documentation for the asset.
+
+ERROR_ASSET_MISSING_DEPENDENT=CWWKF1258E: The {0} asset depends on {1}, which is not available in the configured repositories. Configure installUtility to connect to the IBM WebSphere Liberty Repository to download or install the asset. If you do not have internet access, contact your administrator to download the missing asset and any dependencies and add them to a configured repository.
+ERROR_ASSET_MISSING_DEPENDENT.explanation=The asset cannot be downloaded or installed because it depends on an asset that is not available.
+ERROR_ASSET_MISSING_DEPENDENT.useraction=Configure installUtility to connect to the IBM WebSphere Liberty Repository to download or install the asset. Alternatively, download the wlp-featureRepo-<version>.zip from the IBM Fix Central website, and configure the extracted folder as a directory-based repository. If you do not have internet access, contact your administrator to download the missing asset and any dependencies and add them to a configured repository. For more information about installing assets, see the installUtility documentation in IBM Knowledge Center. If the problem persists, see the problem determination information on the WebSphere Application Server Support web page: http://www.ibm.com/software/webservers/appserv/was/support.
+
+ERROR_FAILED_TO_RESOLVE_ASSETS=CWWKF1259E: Unable to obtain the following assets: {0}. Ensure that the specified assets are valid. To find the IDs of applicable assets, run the installUtility find command.
+ERROR_FAILED_TO_RESOLVE_ASSETS.explanation=The installer could not retrieve the specified assets from the repositories.
```

Comment:
Change "The installer could not retrieve the specified assets from the repositories." to "The installer did not retrieve the specified assets from the repositories." or to "The installer cannot retrieve the specified assets from the repositories."

This is a nit and, if it's not changed, not a problem.